docs/sidebar.json

```diff
@@ -50,6 +50,31 @@
         "cli/kratos-version"
       ]
     }, 
+    {
```

Comment:
why is this here two times now?

Reply:
probably another failure in the automation